### PR TITLE
Demos: improve accessibility ( part 1 )

### PR DESCRIPTION
**Pull request by AlexanderMoiseev**

### CODE REVIEW

**Review by marker-dao — DISMISSED** [review]

File: JSDemos/Demos/DropDownBox/MultipleSelection/Angular/app/app.component.html [hunk 1]
```diff
@@ -35,6 +36,7 @@
     <div class="dx-field-value">
       <dx-drop-down-box
         [(value)]="gridBoxValue"
+        [inputAttr]="{ 'aria-label': Owner' }"
```
Comment (ksercs) [inline JSDemos/Demos/DropDownBox/MultipleSelection/Angular/app/app.component.html]:
```suggestion
        [inputAttr]="{ 'aria-label': Owner' }"
```
```suggestion
        [inputAttr]="{ 'aria-label': 'Owner' }"
```

File: JSDemos/Demos/Scheduler/CurrentTimeIndicator/React/data.js [hunk 2]
```diff
@@ -52,3 +52,5 @@ export const moviesData = [{
   text: 'It\'s a Wonderful Life',
   image: '../../../../images/movies/ItsAWonderfulLife.jpg',
 }];
+
+export const positionLabel = { 'aria-label': 'Positin' };
```
Comment (ksercs) [inline JSDemos/Demos/Scheduler/CurrentTimeIndicator/React/data.js]:
```suggestion
export const positionLabel = { 'aria-label': 'Position' };
```

File: JSDemos/Demos/Common/CustomTextEditorButtons/React/App.js [hunk 3]
```diff
@@ -6,6 +6,12 @@ import { DateBox, Button as DateBoxButton } from 'devextreme-react/date-box';
 
 const millisecondsInDay = 24 * 60 * 60 * 1000;
 
+const currencyLabel = { 'aria-label': 'Multi Currency' };
+
```
Comment (alexanderPolosatov) [inline JSDemos/Demos/Common/CustomTextEditorButtons/React/App.js]: let's remove extra spaces?

File: JSDemos/Demos/Common/CustomTextEditorButtons/React/App.js [hunk 4]
```diff
@@ -6,6 +6,12 @@ import { DateBox, Button as DateBoxButton } from 'devextreme-react/date-box';
...
+const dateBoxLabel = { 'aria-label': 'Date' };
+
```
Comment (alexanderPolosatov) [inline JSDemos/Demos/Common/CustomTextEditorButtons/React/App.js]: let's remove extra space?

File: JSDemos/Demos/TextArea/Overview/React/App.js [hunk 5]
```diff
@@ -7,6 +7,8 @@ import service from './data.js';
 
 const { valueChangeEvents } = service;
```
Comment (alexanderPolosatov) [inline JSDemos/Demos/TextArea/Overview/React/App.js]: let's remove extra space?

File: JSDemos/Demos/DropDownBox/MultipleSelection/Angular/app/app.component.html (hunk 1 above)
Comment (AlexanderMoiseev) [reply JSDemos/Demos/DropDownBox/MultipleSelection/Angular/app/app.component.html]: wow that's a good catch, nice )

**Review by ksercs — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Pull request merged**